### PR TITLE
fix enumerate(Iterable, start=1)

### PR DESCRIPTION
**Pull request by Dustyposa**
fix enumerate(Iterable, start=1) add add test_case 
#954

### CODE REVIEW

File: src/enumerate.js [hunk 1]
```diff
@@ -57,6 +57,11 @@ Sk.builtin.enumerate.prototype.next$ = function (self) {
     return self.tp$iternext();
 };
 
+Sk.builtin.enumerate.co_varnames = ["", "start"];
```
Comment (albertjan) [inline src/enumerate.js]: lets call the first argument "iterable" 😄

**Comment by albertjan** [comment]
Thank you! Looks good! One small comment. 😄

File: src/enumerate.js (hunk 1 above)
Comment (Dustyposa) [reply src/enumerate.js]: > Thank you! Looks good! One small comment.
Ok, I got it, I change the argument will get twice commit, 
Do I need to rebase the branch and make the commit to be once?

Comment (Dustyposa) [reply src/enumerate.js]: > lets call the first argument "iterable"
Ok, I got it, I change the argument will get twice commit,
Do I need to rebase the branch and make the commit to be once?

**Comment by albertjan** [comment]
Merging this as it's a small change.

**Pull request merged**